plugins/org.eclipse.papyrus.uml.interaction.model/src/org/eclipse/papyrus/uml/interaction/model/spi/LayoutConstraints.java

```diff
@@ -41,6 +46,108 @@
 	 */
 	int getYOffset(Compartment shapeCompartment);
 
+	/**
+	 * Queries the minimum height of a view.
+	 * 
+	 * @param view
+	 *            a view.
+	 * @return its minimum height.
+	 */
+	int getHeight(View view);
```

Comment:
This should be `getMinimumHeight` to indicate its meaning.  The `LayoutHelper` has `getHeight` that gets the actual height, and we may in future have `getDefaultHeight` and others ...

Similarly the other methods in this interface.

Reply:
Right, I'll rename those methods.